src/libs/actions/Report.js

```diff
@@ -689,29 +682,25 @@ function readNewestAction(reportID, createdDate) {
 }
 
 /**
- * Sets the last read comment on a report
+ * Sets the last read timestamp on a report
  *
  * @param {String} reportID
- * @param {String} createdDate
- * @param {Number} sequenceNumber
+ * @param {String} created
  */
-function markCommentAsUnread(reportID, createdDate, sequenceNumber) {
-    const newLastReadSequenceNumber = sequenceNumber - 1;
+function markCommentAsUnread(reportID, created) {
+    // We subtract 1 millisecond so that the lastRead is updated to just before this reportAction's created date
```

Comment:
How about something like: 

```
// We subtract 1 millisecond so that the lastRead is updated to just before this reportAction's created date
// For example, if we marked a report action with timestamp 1000 unread, we set the lastRead to 999
// This makes it so that the report action with timestamp 1000 is the first one shown as unread
```

Reply:
Yeah I think that helps visualize it really well.

Reply:
Cool.